posit-dev/positron
# Bundle dark mode icons for the connections pane.

### Intent

> Describe briefly what problem this pull request resolves, or what new feature it introduces. Include screenshots of any new or altered UI. Link to any GitHub issues that are related.

Addresses #2573 
Bundle SVG icons that can be used for dark themes.

### Approach

> Describe the approach taken and the tradeoffs involved if non-obvious; add an overview of the solution if it's complicated.

### QA Notes

> Add additional information for QA on how to validate the change, paying special attention to the level of risk, adjacent areas that could be affected by the change, and any important contextual information not present in the linked issues.

With the IDE in the dark mode, you should see lightly colored icons then looking at connections.

<img width="408" alt="image" src="https://github.com/posit-dev/positron/assets/4706822/f1ff6ff4-38e1-477e-8b89-ab698566ff45">
